include/sound/sof/header.h

```diff
@@ -77,6 +77,7 @@
 #define SOF_IPC_PM_CLK_REQ			SOF_CMD_TYPE(0x006)
 #define SOF_IPC_PM_CORE_ENABLE			SOF_CMD_TYPE(0x007)
 #define SOF_IPC_PM_GATE				SOF_CMD_TYPE(0x008)
+#define SOF_IPC_PM_VFE_POWER_STATUS		SOF_CMD_TYPE(0x010)
```

Comment:
I don't think we need to change the ABI, but the shared file with firmware needs to be updated.
It's also unclear to me if the discontinuity is deliberate or meaningful, if yes you might want to add comments on what the bits represent.

Reply:
@plbossart it is deliberate but we can discuss whether it's meaningful. I have no strong preference here. I tried to follow the pattern used in other IPC groups. Several of them jump to a new 0x10 as an indication of a new sub-group, and those jumps are never documented.

Reply:
@lyakh Can yo umake a FW PR for the addition. It's important to reserve the id, so it doesn't get used for something else.

Reply:
@lyakh we might as well start from BIT(8) to leave some space for 'regular' PM IPCs.

Reply:
@plbossart as agreed with @lgirdwood in https://github.com/thesofproject/sof/pull/2744#issuecomment-611534051 we'll drop this new IPC and re-use `PM_SAVE` and `PM_LOAD` with just adding one more field to the response structure for the latter.